src/client/lazy-app/Compress/Output/style.css

```
@@ -161,3 +166,9 @@ input.zoom {
     pointer-events: auto;
   }
 }
+
+.pixelated {
+  image-rendering: -webkit-optimize-contrast;
+  image-rendering: -moz-crisp-edges;
+  image-rendering: pixelated;
```

```suggestion
  image-rendering: crisp-edges;
  image-rendering: pixelated;
```

Feels better to go with unprefixed versions since they're supported.